packages/build-info/src/package-managers/detect-package-manager.ts

```diff
@@ -43,6 +44,12 @@ export const AVAILABLE_PACKAGE_MANAGERS: Record<PkgManager, PkgManagerFields> =
     lockFile: 'pnpm-lock.yaml',
     forceEnvironment: 'NETLIFY_USE_PNPM',
   },
+  [PkgManager.BUN]: {
```

Comment:
Note: putting it here means if both a `yarn.lock` and `bun.lockb` exist, it'll prefer `yarn` as a package manager (same with pnpm). Since `bun` is relatively new I think that's OK? But let me know if you prefer something else.

Reply:
Thoughts on adding a set of tests for such cases?
- One where we have a `yarn.lock` and a `bun.lockb` and we detect `yarn`
- Same but for pnpm

Reply:
Added!

Reply:
I think the testcase would be:


```
test('should favor yarn over bun if both lockfiles are present', async ({ fs }) => {
  const cwd = mockFileSystem({
    'package.json': '{}',
    'yarn.lock': '',
    'bun.lockb': '',
  })
  const project = new Project(fs, cwd)
  const pkgManager = await detectPackageManager(project)
  expect(pkgManager?.name).toBe('yarn')
})
```

Reply:
I had forgotten to push 🙃 - ended up doing an ammend + force push. Should all be there now.